elated-egrets/bangazon-internal-site-2.0
# Rm products remaining

## DESCRIPTION
added more info to users product view for inventory information

## How to pull down
Commands

```
git fetch --all
git checkout rm-products-remaining
```

## Steps to test

1. Start virtual environment
1. Make sure Django server is running
1. Create some products and put some on orders
1. then navigate to the my products page and see if products sold and remaining are correctly reflected in the info. otherwise functionality here should remain unchanged. 

## What you changed
added additional logic to the list my products view to get data from order items and send that information to the template. 

## What was added to read me

## Final Checklist
Before merging.
1. Has each file changed been reviewed?
2. Has the readme been updated to reflect changes made?
3. Have 2 teammates approved the PR?

Then Ship It!
